integration-cli/docker_cli_rename_test.go

```diff
@@ -136,3 +136,19 @@ func (s *DockerSuite) TestRenameContainerWithLinkedContainer(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(strings.TrimSpace(out), checker.Equals, strings.TrimSpace(db1))
 }
+
+// Test case for #31392
```

Comment:
Can you add a short description of what this test is testing for? It can help if the test is refactored in future and to verify that the test is still testing what it was originally written for 😅 

Also (sorry!) this test will now have to be moved to the `/integration` test-suite, as we're freezing the "old" `integration-cli` test